company/okrs/2020_q2.md

```diff
@@ -90,7 +90,8 @@
           KR: Eliminate flakiness in CI infrastructure and production alerting (OpsGenie). (e.g.,[#9607](https://github.com/sourcegraph/sourcegraph/issues/9607)) \
           KR: Establish clear reporting on utilization of compute resources on sourcegraph.com and internal infrastructure. ([#10101](https://github.com/sourcegraph/sourcegraph/issues/10101))
        1. **Core services: Increase test coverage and reliability.** \
-          KR: Increase Go test coverage from 46% to 60%.
+          KR: Increase Go test coverage from 46% to 60%. \
+          KR: Convert 95% of e2e tests for backend functionality from browser-based to GraphQL-based.
```

Comment:
Can you update these KRs with the current outcome as well?

Reply:
Added!